.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
+install: bundle install --retry=3 --deployment --without development production
 cache: bundler
-sudo: true
+sudo: false
```

Comment:
My 2p: Tonmoy said there are significant efficiencies in not having a `sudo`-enabled Travis environment.  If you don't set `sudo` explicitly to `false` then it presumably will take on whatever default value is currently set by Travis.  If that default happens to change to `true` at some point, we lose these efficiencies by virtue of our Travis environment flipping to a `sudo`-enabled one.

In other words, there is no harm in this case in being explicit in setting `sudo` to `false`, and it may even be beneficial to do so.

Tonmoy presumably can say whether or not `sudo` is a required field in a `travis.yml` file.  Sufia includes it in theirs.